how-to-delete-browser-cookies/script.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * The list of cookies to keep
+ */
+(function () {
+  var itemsToKeep = ["euconsent-v2", "didomi_token"];
+  var itemsToKeepRegex = /some_regex_[a-z0-9]*/;
```

Comment:
something to keep in mind here is that customers can [customize their Didomi cookie names](didomi_token_${regulation}), and we will also begin using one cookie per regulation, with the default pattern `didomi_token_${regulation}`, so for example the default CCPA cookie is called `didomi_token_ccpa`.

Reply:
We need to get actual cookies names form `Didomi.getConfig()` check also `regulation` object